docs/developer-guide.md

```diff
@@ -1 +1,95 @@
 # Developer Guide
+
+This guide will help you get started with developing, deploying, and running Cube AI.
+
+## Cloning the Repository
+
+In order to pull Cube AI Docker images and code from GitHub, you will need to generate a personal access token. Make sure to grant the following permissions to the token:
```

Comment:
This will change since we will be open-sourcing cube AI. I would advise we treat it as if it is a public repo hence, there is no need to generate a token